odoo/http.py

```diff
@@ -1440,6 +1440,7 @@ def dispatch(self, environ, start_response):
             httprequest = werkzeug.wrappers.Request(environ)
             httprequest.app = self
             httprequest.parameter_storage_class = werkzeug.datastructures.ImmutableOrderedMultiDict
+            threading.current_thread().httprequest_url = httprequest.url
```

Comment:
Doesn't this require a corresponding cleanup in `application_unproxied()`?
https://github.com/odoo/odoo/blob/10.0/odoo/service/wsgi_server.py#L157-L165